src/main.c

```diff
@@ -103,30 +113,66 @@ int load_conf_file(char *file_path)
     return 0;
 }
 
+static void run_test(int test_n) {
+    for (int i = 0; i < tests_exclude.size; i++) {
+        // If a match is found in the exclusion list, then we skip the test.
+        if (test_n == vector_get(&tests_exclude, i)) {
+            test_n = -1;
+            break;
+        }
+    }
+    // Skip the test if test_n is a negative number.
+    if (test_n >= 0) {
+        kernel_thunk_table[test_n]();
+    }
+}
+
 static void run_tests()
 {
     print("Random seed used is %u", seed);
     if (tests_to_run.size == 0) {
-        print("No Specific tests specified. Running all tests (Single Pass).");
+        print("No specific tests were requested. Running all tests (Single Pass).");
+        if (tests_exclude.size) {
+            int remainder_size = 0;
```

Comment:
Nit: this variable does not really contain a remainder size. It contains an exclusion count in this code block.

Reply:
It's best to keep the wording different since `tests_exclude.size` and `(tests_)exclude_size` are too identical to cause confusion. Though, `exclude_count` can work as a replacement.